### PR TITLE
Added pytest.yml for configuring GitHub actions

### PR DESCRIPTION
**Pull request by meenakshidembi691**
Added pytest.yml for configuring GitHub actions

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Run Unit Test via Pytest  
+  
+on: [push]  
+  
+jobs:  
+  build:  
+    runs-on: ubuntu-latest  
+    strategy:  
+      matrix:  
+        python-version: ["3.10"]
```
Comment (Jennifer-John) [inline .github/workflows/pytest.yml:10]: should we include all supported python versions?

File: .github/workflows/pytest.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    steps:  
+      - uses: actions/checkout@v3  
+      - name: Set up Python ${{ matrix.python-version }}  
+        uses: actions/setup-python@v4  
+        with:  
+          python-version: ${{ matrix.python-version }}  
+      - name: Install dependencies  
+        run: |  
+          python -m pip install --upgrade pip
+          pip install testtools
```
Comment (Jennifer-John) [inline .github/workflows/pytest.yml:21]: if these are part of requirements.txt, can we remove the explicit pip install here?

File: .github/workflows/pytest.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          pip install requests
+          pip install pytest
+          pip install pytest-coverage
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Test with pytest  
+        run: |  
+          python -m unittest -v tests/test_*.py
```
Comment (Jennifer-John) [inline .github/workflows/pytest.yml:28]: since we are installing pytest, shud we use -m pytest?

**Review by felixs88 — APPROVED** [review]

File: .github/workflows/pytest.yml (hunk 2 above)
Comment (meenakshidembi691) [reply .github/workflows/pytest.yml:21]: will remove these ones requirements.txt PR is merged

File: .github/workflows/pytest.yml (hunk 3 above)
Comment (meenakshidembi691) [reply .github/workflows/pytest.yml]: Running the tests with pytest needs changes in the directory structure, hence continuing with python unittest

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**